tests/tests.cmake

```diff
@@ -201,4 +203,4 @@ target_sources(testprecice
     )
 
 # Contains the list of integration test suites
-set(PRECICE_TEST_SUITES Parallel QuasiNewton Serial)
+set(PRECICE_TEST_SUITES GeomultiscaleMapping Parallel QuasiNewton Serial)
```

Comment:
What is actually the motivation for having a test suite specifically for the geometric multiscale? Other mappings are part of the serial and parallel test suites.

I do not disagree, but we may need some documentation and reconsideration here.

Reply:
That was Frederic's wish, because apparently the serial and parallel test suites are outdated.
![image](https://github.com/precice/precice/assets/99761626/2363171b-e565-4a5a-81ec-34a7992b0b35)